patches/0010-LPS-139565-When-upgrading-from-6.2-to-7.1-image-widt.patch

```diff
@@ -0,0 +1,53 @@
+From 041396d0d9997e8c078eb17a50f7348a4d2a64e8 Mon Sep 17 00:00:00 2001
+From: Minhchau <minhchau.dang@liferay.com>
+Date: Tue, 28 Sep 2021 11:18:40 -0700
+Subject: [PATCH] LPS-139565 When upgrading from 6.2 to 7.1, image width/height
+ in web content is lost
+
+Move inline styles for height/width to image attributes
+---
+ plugins/image2/dialogs/image2.js | 16 ++++++++++++++++
+ plugins/image2/plugin.js         |  3 +++
+ 2 files changed, 19 insertions(+)
+
+diff --git a/plugins/image2/dialogs/image2.js b/plugins/image2/dialogs/image2.js
+index b09a05f15..0782fa2e6 100644
+--- a/plugins/image2/dialogs/image2.js
++++ b/plugins/image2/dialogs/image2.js
+@@ -394,6 +394,22 @@ CKEDITOR.dialog.add( 'image2', function( editor ) {
+ 			// Create a "global" reference to widget's image.
+ 			image = widget.parts.image;
+ 
++			var styleHeight = image.$.style.removeProperty('height');
++
++			if (styleHeight) {
++				styleHeight = styleHeight.replace('px', '');
++				widget.data.height = styleHeight;
++				image.$.setAttribute('height', styleHeight);
++			}
++
++			var styleWidth = image.$.style.removeProperty('width');
++
++			if (styleWidth) {
++				styleWidth = styleWidth.replace('px', '');
++				widget.data.width = styleWidth;
++				image.$.setAttribute('width', styleWidth);
++			}
++
```

Comment:
While this code does the job, it is not placed in the ideal workflow spot. We are reading properties, as well as mutating underlying HTML in `onShow`, where I would not expect this logic to happen.

We can define [the internal data.width](https://github.com/ckeditor/ckeditor4/blob/master/plugins/image2/plugin.js#L378-L379) to read from `element.style`, with something like  
```js
data = {
	width: image.$.style.width && parseInt(image.$.style.width, 10) || image.getAttribute( 'width' ) || '',
	...
}
```
Then, we can cleanup HTML [on save](https://github.com/ckeditor/ckeditor4/blob/master/plugins/image2/plugin.js#L1117), with something like:
```js
if ( dimensions[ d ] )
	image.setAttribute( d, dimensions[ d ] );
	
	image.$.style.removeProperty(d);
```

Reply:
@markocikos I've applied the requested changes.